dbms/tests/queries/0_stateless/01050_window_view_sql_parser_tumble.sql

```diff
@@ -0,0 +1,32 @@
+SET allow_experimental_window_view = 1;
+
+DROP TABLE IF EXISTS test.mt;
+
+CREATE TABLE test.mt(a Int32, timestamp DateTime) ENGINE=MergeTree ORDER BY tuple();
+
+SELECT '---WATERMARK---';
+DROP TABLE IF EXISTS test.wv;
```

Comment:
`clickhouse-test` uses random database each run, so hard coding here `test` database should be avoided (and you can simply pass `--database test` to archive the same database between test runs)